javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,72 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  // You write code here
+  if (studentName === "") {
+    return console.log(`Please enter the student name.`);
+  }
+  for (i = 0; i < class07Students.length; i++) {
+    if (class07Students[i] === studentName) {
+      return console.log(
+        `This ${studentName} student is already in the class.`
+      );
+    }
+  }
+  if (studentName.substring(0, 5) === "Queen") {
+    console.log(`Welcome to class Queen`);
+    return class07Students.push(studentName);
+  } else if (class07Students.length === 6) {
+    return console.log(`Cannot add more students to class 07.`);
+  } else if (class07Students.length < 6) {
+    return class07Students.push(studentName);
+  }
+}
```

Comment:
`return console.log(...)` returns `undefined`, you output is never captured in any variable below. and when you push to the array the function access the `class07Students` from the global scope. You basically dont need any `return`s in this function

Reply:
understood!!